Tests/JWTTests/Helper/throwsError.swift

```diff
@@ -1,10 +1,12 @@
 @testable import JWT
 import XCTest
 
-func assert<E: Error>(_ expression: @autoclosure () throws -> Any,
+func assert<E: Error>(
```

Comment:
This function is not used anymore, right? Since you changed the code at the places this used to be called. Not a fan of fancy test helper functions? ;)